common/sgx/qeidentity.c

```diff
@@ -120,14 +133,24 @@ oe_result_t oe_enforce_qe_identity(sgx_report_body_t* qe_report_body)
         &pck_cert_chain));
     OE_TRACE_INFO("oe_verify_ecdsa256_signature succeeded\n");
 
+    // Get leaf certificate
+    OE_CHECK_MSG(
+        oe_cert_chain_get_leaf_cert(&pck_cert_chain, &leaf_cert),
+        "Failed to get leaf certificate.",
+        NULL);
+    oe_cert_get_validity_dates(&leaf_cert, &from, &until);
+
+    oe_datetime_log_info("QE identity cert issue date: ", &from);
+    oe_datetime_log_info("QE identity cert next update: ", &until);
+
     // Check that issue_date and next_update are after the earliest date that
     // the enclave accepts.
     if (oe_datetime_compare(
-            &parsed_info.issue_date, &_sgx_minimim_crl_tcb_issue_date) != 1)
+            &parsed_info.issue_date, &_sgx_minimim_crl_tcb_issue_date) < 0)
         OE_RAISE(OE_INVALID_QE_IDENTITY_INFO);
```

Comment:
Why not use ON_RAISE_WITH_MSG here to make it easier to pin point the error source?

Reply:
Done.